wwwroot/index.php

```diff
@@ -409,10 +416,15 @@ function PageTitler($Page)
         }
 
         $PageTitle = PageTitler($PageTitle);
-        $Content['Title'] .= 'Editing: '.FishFormat($PageTitle);
+        if (!empty($_POST)) {
+            // code...
+            $Content['Title'] .= 'Editing: '.FishFormat($PageTitle);
+        }
+        
```

Comment:
This logic breaks the title when editing a page

![image](https://github.com/wetfish/wiki/assets/1670549/434216e8-06f2-453c-a68e-fec9a48ca17a)